pycode/epidemiology/epidata/getDIVIData.py

```diff
@@ -602,6 +602,15 @@ def get_divi_data(read_data=dd.defaultDict['read_data'],
             [dd.EngEng["ICU"], dd.EngEng["ICU_ventilated"]],
             impute='forward', moving_average=moving_average)  
 
+    # add names etc for empty frames (counties where no ICU beds are available)
+    countyid_to_name = geoger.get_countyid_to_name()
+    stateid_to_name = geoger.get_stateid_to_name()
+    countyid_to_stateid = geoger.get_countyid_to_stateid_map()
+    for id in df.loc[df.isnull().any(axis=1), dd.EngEng['idCounty']].unique():
+        stateid = countyid_to_stateid[id]
+        df.loc[df[dd.EngEng['idCounty']] == id, [dd.EngEng['idState'], dd.EngEng['state'],
+                                                 dd.EngEng['county']]] = [stateid, stateid_to_name[stateid], countyid_to_name[id]]
```

Comment:
In the data from the new source there dfare no empty frames, so this step is not needed anymore.

Reply:
I dont think this is about empty frames but about missing entries in rows where the county id is given. This may be useful or may be not. If in the following an aggregation etc based on state_ids etc is done, it would probably give an error if N/A is found there.

Maybe it should even be discussed if this precheck is not necessary for all data frames that we download?!